includes/class-comment-walker.php

```diff
@@ -170,4 +169,115 @@ protected function avatar_only( $comment, $depth, $args ) {
+		$url = get_url_from_webmention( $comment );
+		$host = wp_parse_url( $url, PHP_URL_HOST );
+		$host = preg_replace( '/^www\./', '', $host );
+		$type = get_webmention_comment_type_attr( $comment->comment_type, 'class' );
+		if ( 'comment' === $comment->comment_type ) {
+			$type = 'p-comment';
+		}
+
+		$commenter          = wp_get_current_commenter();
+		$show_pending_links = ! empty( $commenter['comment_author'] );
+
+		if ( $commenter['comment_author_email'] ) {
+			$moderation_note = __( 'Your comment is awaiting moderation.', 'default' );
+		} else {
+			$moderation_note = __( 'Your comment is awaiting moderation. This is a preview; your comment will be visible after it has been approved.', 'default' );
+		}
+		?>
+		<<?php echo $tag; ?> id="comment-<?php comment_ID(); ?>" <?php comment_class( $this->has_children ? 'parent' : '', $comment ); ?>>
+			<article id="div-comment-<?php comment_ID(); ?>" class="comment-body h-cite <?php echo $type; ?>">
+				<footer class="comment-meta">
+					<div class="comment-author vcard h-card u-author">
+						<?php
+						if ( 0 != $args['avatar_size'] ) {
+							echo get_avatar( $comment, $args['avatar_size'] );
+						}
+						?>
+						<?php
+						$comment_author = get_comment_author_link( $comment );
+
+						if ( '0' == $comment->comment_approved && ! $show_pending_links ) {
+							$comment_author = get_comment_author( $comment );
+						}
+
+						printf(
+							/* translators: %s: Comment author link. */
+							__( '%s <span class="says">says:</span>', 'default' ),
+							sprintf( '<b class="fn">%s</b>', $comment_author )
+						);
+						if ( ! empty( $cite ) && 'webmention' === get_comment_meta( $comment->comment_ID, 'protocol', true ) ) {
+							printf( $cite, $url, $host );
+						}
+
+						?>
+					</div><!-- .comment-author -->
+
+					<div class="comment-metadata">
+						<?php
+						// Allow arbitrary additions to comment metadata.
+						do_action( 'webmention_comment_metadata', $comment );
+						printf(
+							'<a class="u-url" href="%s"><time class="dt-published" datetime="%s">%s</time></a>',
+							esc_url( get_comment_link( $comment, $args ) ),
+							get_comment_time( DATE_W3C ),
+							sprintf(
+								/* translators: 1: Comment date, 2: Comment time. */
+								__( '%1$s at %2$s' ),
+								get_comment_date( '', $comment ),
+								get_comment_time()
+							)
+						);
+
+						edit_comment_link( __( 'Edit', 'default' ), ' <span class="edit-link">', '</span>' );
+						?>
+					</div><!-- .comment-metadata -->
+
+					<?php if ( '0' == $comment->comment_approved ) : ?>
+					<em class="comment-awaiting-moderation"><?php echo $moderation_note; ?></em>
+					<?php endif; ?>
+				</footer><!-- .comment-meta -->
+
+				<div class="comment-content e-content p-name">
+					<?php comment_text(); ?>
+				</div><!-- .comment-content -->
+
+				<?php
+				if ( '1' == $comment->comment_approved || $show_pending_links ) {
```

Comment:
```suggestion
				if ( '1' === $comment->comment_approved || $show_pending_links ) {
```

Reply:
Oddly, all of the == vs === are straight out of the original html5_comment.